tests/test_config.py

```diff
@@ -764,6 +764,43 @@ def test_get_audience__invalid_id(self):
 
     self.assertIsNone(self.project_config.get_audience('42'))
 
+  def test_get_audience__retrieves_audiences_first_from_typedAudiences_before_audiences(self):
+    opt = optimizely.Optimizely(json.dumps(self.config_dict_with_typed_audiences))
+    config = opt.config
+
+    audiences = self.config_dict_with_typed_audiences['audiences']
+    typed_audiences = self.config_dict_with_typed_audiences['typedAudiences']
+
+    audience_3988293898 = {
+      'id': '3988293898',
+      'name': '$$dummySubstringString',
+      'conditions': '{ "type": "custom_attribute", "name": "$opt_dummy_attribute", "value": "impossible_value" }'
+    }
+
+    self.assertTrue(audience_3988293898 in audiences)
+
+    typed_audience_3988293898 = {
+      'id': '3988293898',
+      'name': 'substringString',
+      'conditions': ['and', ['or', ['or', {'name': 'house', 'type': 'custom_attribute',
+                     'match': 'substring', 'value': 'Slytherin'}]]]
+    }
+
+    self.assertTrue(typed_audience_3988293898 in typed_audiences)
```

Comment:
The actual test case assertion is below this. 
The above two are just to show that these audiences **do** exist in the datafile.

Reply:
Ahh, I see, you're checking `self.config_dict_with_typed_audiences['audiences']`/`self.config_dict_with_typed_audiences['typedAudiences']` rather than `opt.config.audiences`/`opt.config.typed_audiences`. Seems fine, though it might be nice to comment that this is just a sanity check for our tests